lighthouse-extension/app/src/popup.js

```diff
@@ -133,11 +133,16 @@ document.addEventListener('DOMContentLoaded', _ => {
       background.createPageAndPopulate(results);
     })
     .catch(err => {
-      let {message} = err;
-      if (err.message.toLowerCase().startsWith('another debugger')) {
+      let message = err.message;
+      if (message.toLowerCase().startsWith('another debugger')) {
         message = 'You probably have DevTools open.' +
           ' Close DevTools to use lighthouse';
       }
+      if (message.toLowerCase().startsWith('unable to kill')) {
+        message = 'You probably have multiple tabs open.' +
```

Comment:
`You probably have multiple tabs open to the same origin` or something like that?